react-components/src/analysis/ids.js

```diff
@@ -1,63 +1,70 @@
-export default ({
-    CHECKBOX: ".checkbox",
-    NAME: "name",
-    OWNER: "owner",
-    APP: "app",
-    START_DATE: "start_date",
-    END_DATE: "end_date",
+export default {
+  CHECKBOX: ".checkbox",
+  NAME: "name",
+  OWNER: "owner",
+  APP: "app",
+  START_DATE: "start_date",
```

Comment:
Just picking a random spot - did the indentation change from 4 spaces to 2?  It seems like all the *.js files in this PR now have tighter spacing.

Reply:
I think IntelliJ applied some Prettier formatting 😄. I will rebase on `master` and format it one more time just to be sure.